### PR TITLE
Allow zooming in/out with numpad +/-

### LINKED ISSUE #570: Ctrl +/- for zooming does not work
The shortcut does not work on Mac as well as on Windows.

### PR DESCRIPTION
**Pull request by BeckmaR**
Resolves #570

### CODE REVIEW

**Comment by BeckmaR** [comment]
@tkutz: Please test on OS/X
@jdicks: Please test on Windows

File: plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/editor/StatechartDiagramEditor.java [hunk 1]
```diff
@@ -10,6 +10,8 @@
  */
 package org.yakindu.sct.ui.editor.editor;
 
+import java.awt.event.KeyEvent;
+
```
Comment (tkutz) [inline plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/editor/StatechartDiagramEditor.java]: This is probably unused

**Comment by BeckmaR** [comment]
Hey @tkutz please review again

**Comment by jdicks** [comment]
tested on windows also, works fine

**Pull request merged**